tests/test_handlers.py

```diff
@@ -604,7 +612,7 @@ def generate_segments(self) -> None:
                 offset += bytes_read
                 self.segment_idx += 1
             self.segment_path_list.append(segment_path)
-            if meta == None or bytes_read == 0:
+            if meta is None or bytes_read == 0:
```

Comment:
Can we simply code to
```
not meta or not bytes_read
or
not meta or bytes_read == 0
```

Reply:
Same to the previous comments, mypy will enforce to check if meta is "None". Simplify the code will fail the mypy checking.